recipes/fish-shell/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+export   CFLAGS="-I$PREFIX/include -I$PREFIX/include/ncursesw"
```

Comment:
That was thought for vertical alignment, but that probably isn't any use here. I'll remove the extra spaces.